packages/ledger/src/types/ledger.responses.ts

```diff
@@ -2,19 +2,19 @@ import type { Principal } from "@dfinity/principal";
 import type { Subaccount, Value } from "../../candid/icrc1_ledger";
 
 // Source: https://github.com/dfinity/icrc-1#standard-metadata-entries
-export enum SnsMetadataResponseEntries {
+export enum Icrc1MetadataResponseEntries {
   SYMBOL = "icrc1:symbol",
   NAME = "icrc1:name",
   DECIMALS = "icrc1:decimals",
   FEE = "icrc1:fee",
 }
 
-export type SnsTokenMetadataResponse = [
-  string | SnsMetadataResponseEntries,
+export type Icrc1TokenMetadataResponse = [
+  string | Icrc1MetadataResponseEntries,
   Value
 ][];
 
-export interface SnsAccount {
+export interface Icrc1Account {
```

Comment:
Shall we ask the FI team whether this is the same for icrc-2 as well?

Reply:
According this doc (https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-2/README.md) there are same. So will rename to `IcrcAccount`